packages/core/util/util.js

```diff
@@ -191,7 +191,7 @@ const memoize = (fn) => {
 		if (n in cache) {
```

Comment:
should we use a different value for the `key` instead of `args[0]`? maybe `JSON.stringify(args)`?

Reply:
I don't think we want to go so far to stringify the args. That could be an expensive operation, particularly if used with `handlers` which receive the props and context objects. We may want a more robust memoization function (likely using a third party module) in the future but I'm okay with this implementation for now.